CMP1903_A1_2324/Testing.cs

```diff
@@ -16,5 +17,24 @@ internal class Testing
          */
 
         //Method
+        //Tests each roll and sum to see if the die object's values match the values recorded in the game class
+        //Should come up with no error messages if all values match
+        public void TestGame()
+        {
+            //Tells user that they are now on testing run
+            Console.WriteLine("Now on the testing run....");
+
+            //Creates new game object
+            Game testGame = new Game();
+            testGame.PlayGame();
+
+            //Sees if each die value matches their recorded roll
+            Debug.Assert(testGame.one.Value == testGame.roll1);
+            Debug.Assert(testGame.two.Value == testGame.roll2);
+            Debug.Assert(testGame.three.Value == testGame.roll3);
```

Comment:
Could be beneficial to include a custom message incase one of these Debug.Asserts return a false/invalid answer, easier to identify an error within a specific operation.

Debug.Assert(condition, "Custom Error Message")